SU-SWS/stanford_basic
# D8CORE-2008: Removing the secondary skip link when the hamburger visible

# READY FOR REVIEW

# Summary
- When you are on a page with secondary navigation and the mobile responsive hamburger menu is visible, remove the skip to secondary navigation link because it does nothing.
- **NOTE**: it doesn't respond to when you resize the browser window. @cjwest mentioned she thought it was ok since if you are using a screen reader you won't be resizing the window. Is this the best solution? Or should it be responsive? My thought was since it was for keyboard and screenreaders and not visible otherwise, this seemed better. The other option would be to track the window size changes.

# Review By (Date)
- 5/15

# Urgency
- med

# Steps to Test

1. Pull in this change.
2. Clear caches
3. Create a secondary navigation.
4. Shrink the browser so you have a hamburger menu like this:
5. <img width="961" alt="Screen Shot 2021-05-18 at 7 53 37 AM" src="https://user-images.githubusercontent.com/1187335/118673979-4b45f300-b7ae-11eb-8031-7d22d8234f35.png">
5.  Go to the page that has a secondary navigation menu on it.  Press tab to see only the `Skip to main content`. 
6. You should not see this:
7. <img width="951" alt="Screen Shot 2021-05-18 at 7 57 57 AM" src="https://user-images.githubusercontent.com/1187335/118674785-e9d25400-b7ae-11eb-86a3-f1c4cd8c5cdc.png">
8. The idea is that if you have the hamburger menu you shouldn't get that `Skip to Secondary Menu` link because it is a useless link.
9. In the above note, I tell about the limitations of this.


# Affected Projects or Products
- Stanford Basic theme

# Associated Issues and/or People
- D8CORE-2008
- @cjwest 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
